src/app/search/search-entry-table.ts

```diff
@@ -33,6 +33,32 @@ export abstract class SearchEntryTable extends Base implements OnInit {
 
   public readonly displayedColumns = ['name', 'verified', 'author', 'descriptorType', 'projectLinks', 'starredUsers'];
   abstract dataSource: MatTableDataSource<Workflow | DockstoreTool>;
+
+  static compareAttributes(a: DockstoreTool | Workflow, b: DockstoreTool | Workflow, attribute: string, direction: string) {
```

Comment:
Should have a unit test for this

Reply:
FWIW, it will be easier to unit test if it's in the service, at least it is for me. I dislike writing unit tests for components because there's so much more going on, whereas with a service you can just have a function you can invoke and test the outputs.